notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb

```diff
@@ -0,0 +1,655 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.        ax.axvline(pd.to_datetime(reference_t), lw="0.5", c="k")
</pre><p> </p><p>I don't get what the black line is indicating here. Is it indicating the azimuth mount behavior after a slew? Was this a azimuth slew from 0 to ~3.5, while the mount should remain stationary in elevation (but we actually see some variation?</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/94/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
This is an auxiliary function mainly to illustrate the mount motion together with the IMS position measurements. The black line shows the slew.end according to the EventMaker. Added this info in a legend in this auxiliary plot.

Reply:
Perfect!